src/ralph/api/routers/health.py

```diff
@@ -32,7 +35,7 @@ async def heartbeat():
 
     Returns a 200 if all checks are successful.
     """
-    content = {"database": DATABASE_CLIENT.status().value}
+    content = {"database": await_if_coroutine(BACKEND_CLIENT.status()).value}
```

Comment:
shouldn't this be awaited ?

If yes shouldn't this be breaking a test ?

Reply:
Indeed it should be awaited, good catch!